### ISSUE TITLE
Key `Space` decelerates all players

### ISSUE THREAD

**Issue by EvoLandEco** [tresinformal/game#540]
Occasionally we found that pressing `Space` decelerates all the players. I didn't find anything if I search for `space` throughout the code. `Brake` does not exist in the latest develop, nor do the key action map between `Space` and any action. Any idea about this?

**Comment by ollyturner** [comment]
It actually accelerated, however that may just be a problem with my computer

**Comment by ollyturner** [comment]
I can test it again just to be sure

**Comment by EvoLandEco** [comment]
> I can test it again just to be sure

That's true. The actual behavior is acceleration, but on the fourth panel you can observe that the max speed is reduced. 😿

**Comment by ollyturner** [comment]
Yes I think that was it
